cmd/osquery-perf/agent.go

```diff
@@ -63,30 +62,39 @@ func init() {
 type Stats struct {
 	errors            int
 	enrollments       int
+	orbitenrollments  int
 	distributedwrites int
+	orbitErrors       int
+	desktopErrors     int
 
 	l sync.Mutex
 }
 
-func (s *Stats) RecordStats(errors int, enrollments int, distributedwrites int) {
+func (s *Stats) RecordStats(errors, enrollments, orbitenrollments, distributedwrites, orbitErrors, desktopErrors int) {
```

Comment:
I think this may be a good opportunity to refactor this function. It's pretty hard to determine which stat is being incremented without looking up the function signature. I would suggest creating separate functions for each ie `IncrementErrors`, `IncrementEnrollments`, etc

Reply:
I was torn between pushing this to a follow-up PR, but I decided to bite the bullet now. Updated!